UnderscoreWrapper.cfc

```diff
@@ -0,0 +1,34 @@
+component displayname="UnderscoreWrapper" {
+
+	property name="obj";
+	property name="_";
+
+	public UnderscoreWrapper function init(any obj = {}, _ = new Underscore()) {
+		variables.obj = arguments.obj;
+		variables._ = arguments._;
+
+		return this;
+	}
+
+	public any function value() {
+		return variables.obj;
+	}
+
+	public UnderscoreWrapper function onMissingMethod(required string missingMethodName, required any missingMethodArguments) {
+		var methodToCall = variables._[arguments.missingMethodName];
```

Comment:
It might be good to add a test here for methods that are not defined in Underscore.cfc? (In case someone is actually trying to access a missing method)